projects/demo/src/modules/components/block-status/examples/4/index.html

```diff
@@ -10,10 +10,10 @@
             <tui-avatar
                 *ngFor="let user of users"
                 size="l"
-                [autoColor]="true"
-                [rounded]="true"
-                [text]="user"
-            ></tui-avatar>
+                [style.background]="user | tuiAutoColor"
+            >
+                {{ user | tuiInitials }}
```

Comment:
It is meant to be `[src]="user | tuiInitials"` as per design font is a bit bigger that way.

Reply:
fixed